Migration.Toolkit.CLI/appsettings.json

```diff
@@ -23,6 +23,14 @@
         "CMSConnectionString": "[TODO]"
       }
     },
+    "OptInFeatures":{
+      "QuerySourceInstanceApi": {
+        "Enabled": false,
+        "Connections": [
+          { "SourceInstanceUri": "[TODO]", "SourceInstanceSiteName": "[TODO]", "Secret": "[TODO]" }
```

Comment:
Please remove this property. It should be in sync with siteId from command prompt.